imap_processing/cdf/tests/test_cdf_attribute_manager.py

```diff
@@ -152,6 +156,35 @@ def test_global_attribute():
         == cdf_manager.global_attributes["imap_test_T1_test"]["Data_type"]
     )
 
+    # Testing second elif statement
+    # Should throw error
+
+    # Load in more data using get_global_attributes
+    test_get_global_attrs_2 = cdf_manager.get_global_attributes("imap_test_T2_test")
+    # Testing information previously loaded into global attributes
+    assert test_get_global_attrs_2["Project"] == "STP>Solar-Terrestrial Physics"
+    assert (
+        test_get_global_attrs_2["Source_name"]
+        == "IMAP>Interstellar Mapping and Acceleration Probe"
+    )
+    assert test_get_global_attrs_2["Mission_group"] == "Dysfunctional Cats"
+
+    # Testing if statement
+    assert test_get_global_attrs_2["Descriptor"] == "TEST>Testinstrument"
+    # "Data_type" not required according to default schema
+    assert test_get_global_attrs_2["Data_type"] == "T2_test-two>Test-2 test two"
+    assert test_get_global_attrs_2["Logical_source"] == "imap_test_T2_test"
+    assert (
+        test_get_global_attrs_2["Logical_source_description"]
+        == "IMAP Mission TEST two document Level-T2."
+    )
+    assert test_get_global_attrs_2["LINK_TEXT"] == "Test two additional info"
+
+    # Trying to update a default global using get_global_attributes does not work.
+    # For example, thing about DOI event.
+
+    # END END END
```

Comment:
You can use things like `# ============` to split up sections. But, if you're splitting sections, you can also consider putting it into a different test. It's useful to have small tests because if `test_adding_second_global_attribute_file` fails you know exactly what happened, whereas `test_global_attribute` isn't as clear